blips/blip-3-peerreview.md

```diff
@@ -0,0 +1,53 @@
+This BLIP is a draft: the collaborative document we use to write it is https://hackmd.io/QtoxlUimQNSJByIRKm0IwQ?edit
+
+# Peer Review
+
+| BLIP:     | 3                                                        |
+| -------- | ------------------------------------------------------------ |
+| Title:   | Peer Review                                      |
+| Owner(s):  |                                                            |
+| Author(s):  | Antonio Tenorio Fornés (antonio@decentralized.science), Kaan Uzdogan (uzdogan@mpdl.mpg.de), Paulo Colombo (pcolombo@ucm.es), Elena Pérez Tirador (elena@decentralized.science)                                                           |
+| Status:  | ![Draft](http://rfc.unprotocols.org/spec:2/COSS/draft.svg) |
+| Created: | 2020-06-09                                                   |
+| License: | BSD-2-Clause                                                 |
+
+A draft for a standardized peer review format
+
+## Standard Peer Review registries
+Currently, peer-review-app is registering many fields of a peer review such as author of the review, or the ID of the reviewed article.
+
+This standard should specify which fields to register in every record, the format and standards those fields should adopt, which are mandatory among others.
+
+
+### Peer Review Records' fields
+> [name=Paulo] What is the `id` field and why do we need it ?
+
+| Field | data type | Format/s | Mandatory |  Comments|
+| -------- | ---- | ---- | -------- |----|
+|     id | string  | IPFS Hash??   | Yes     | The id should be unique. How to make sure reviews are unique? See [Zooko's triangle](https://en.wikipedia.org/wiki/Zooko%27s_triangle) for a relevant discussion on the compromises between decentralization, human usability and unique identifiers   |
+|     timestamp | uint32     | unix timestamp     |  Yes    |
+|     author | address     |      |  Yes    |
+|     journalId | string     |      |  Yes    |
+|     publisher | string     |      |  Yes    |
+|     manuscriptId | string     |     |  Yes    |
+|     manuscriptHash | string     |     |  Yes    |
+|     recommendation | enum     |     |  Yes    |
+|     url | string     |     |  Yes    | The url should give public access to the content of the review. Alternativelly, it can be a IPFS address so the content is served from a decentralized network
+|     verified | bool     |     |  Yes    |
```

Comment:
When does the review become verified and who may set this value? Or can this also be a transient value, e.g.
```
isVerified() {
  return vouchers.size > n
}
```

Reply:
I guess different applications and users would like to decide their criteria independently.

Currently, only one "vouche" was needed. In a more decentralized fashion, each actor would decide depending on which identities vouched for the review (stored at vouchersMap)

For instance, our app could only trust reviews verified by us, while other software could require a minimum number, or trust only a collection of actors.

Reply:
Makes sense, but what was the meaning of the `verified` field in this case?

Reply:
I would deprecate that field. We added it because current implementation has it.